src/pulselistener/pulselistener/lib/utils.py

```diff
@@ -116,7 +141,7 @@ def batch_checkout(repo_url, repo_dir, revision=b'tip', batch_size=100000):
 
 def robust_checkout(repo_url, repo_dir, branch=b'tip'):
     '''
-    Helper to clone Mozilla Central using the robustcheckout extension
+    Helper to clone a mercurial repo using the robustcheckout extension
```

Comment:
Nit: I guess this change belongs to another commit :P

Reply:
Oops.